src/Catalog/sparql/ConstructRegistrationPageContentGraph.rq

```diff
@@ -47,6 +47,8 @@ WHERE
 
 	BIND(@licenseUrl AS ?licenseUrl)
 
+	BIND(@iconUrl AS ?iconUrl)
```

Comment:
So it will generate an empty url string for icon like licenseUrl if no catalog icon info?

Reply:
This just binds `?iconUrl` to whatever code produces and it produces an empty string in the case you describe, so yes.